osu.Game/Screens/Select/Carousel/DrawableCarouselBeatmap.cs

```diff
@@ -102,37 +99,26 @@ private void load(BeatmapManager? manager, SongSelect? songSelect)
 
             Header.Children = new Drawable[]
             {
-                new Container
+                new BufferedContainer
```

Comment:
![osu! 2023-01-23 at 04 54 27](https://user-images.githubusercontent.com/191335/213968248-9db84e07-1f66-4442-b4f8-c8f85b56056d.png)

Probably want to avoid this bleed?